.github/workflows/run-tests.yml

```diff
@@ -1,14 +1,14 @@
 name: Run Gradle tests
 
-on: push
+on: [push, pull_request]
```

Comment:
Not sure, but it might be that we will get double builds, one for pr and one for push. I think it is usually done like: on any pr or on push to master

Reply:
Fixed